tests/calculators/refl1d/test_refl1d_wrapper.py

```diff
@@ -276,3 +282,137 @@ def test_sld_profile(self):
         p.add_item('Item', 'MyModel')
         assert_almost_equal(p.sld_profile('MyModel')[1][0], 0)
         assert_almost_equal(p.sld_profile('MyModel')[1][-1], 4)
+
+
```

Comment:
Just a comment about the dangers of comparing equality of floats in the below tests

Reply:
I am not sure what you are referring to here

Reply:
I was just referring to the fact that all the tests are using `==` with floating point numbers, and if any manipulations are done to those numbers, it would likely break (since `0.3 != (0.1+0.2)`). I guess that the tests are all either pointing at the same memory block, or not altering the arrays in this case though.